include/zephyr/drivers/spi.h

```diff
@@ -329,8 +354,10 @@ struct spi_config {
 	 * if not used).
 	 */
 	struct spi_cs_control cs;
-};
 
+	/** @brief Delay between data frames in nanoseconds (0 means 50% of frequency) */
+	uint16_t dfs_delay_ns;
```

Comment:
and this could go into the anonymous struct you added, in the union.

That way: to change in structure size.

That said, this is very much spi controller specific. Some controller do not offer anything to configure this. 
And 0 as 50% of the frequency, why? why not just "hardware default"?

Reply:
I thought about this too but since the struct was called spi_cs_control and this was not related I thought it would be confusing